Makefile

```diff
@@ -0,0 +1,54 @@
+SLEEP = 60
+DOCKER_COMPOSE_DEV = docker-compose-dev.yml
+DOCKER_TEST = docker exec -i anodot-agent pytest
+
+all: build test-all
+
+build:
+	docker-compose up -d --build
+	sleep $(SLEEP)
+	./setup_tests.sh
+
+test-all:
+	$(DOCKER_TEST) -x
+
+all-dev: build-dev test-all-dev
+
+build-dev:
+	docker-compose -f $(DOCKER_COMPOSE_DEV) up -d --build
+	docker exec -i anodot-agent python setup.py install
+	sleep $(SLEEP)
+	./setup_tests.sh
+
+test-all-dev:
+	$(DOCKER_TEST) -x
```

Comment:
Before executing tests in the dev environment you need to clean up the state. Add `docker-compose down -v` and `docker-compose up -d`. For separate source tests in the `up` command specify only services that needed to execute the test. For example, for kafka you need agent, dc, Kafka and zookeeper, for elastic you need agent, dc and es.